### PR TITLE
fix(Action): Labeler

### PR DESCRIPTION
**Pull request by Kitzunu**
## CHANGES PROPOSED:
-  Changing the action to a better one to hopefully get a working labeler with good exceptions

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+DB:
+  - src/**/*.sql
+
+CORE:
+  - src/**/*.cpp
+  - src/**/*.h
+  - '!src/server/scripts/*'
```
Comment (FrancescoBorzi) [inline .github/labeler.yml:7]: are you sure it shouldn't be `'!src/server/scripts/**` ? (with double `*`)

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Script:
+  - src/server/scripts/*.cpp
+  - src/server/scripts/*.h
```
Comment (FrancescoBorzi) [inline .github/labeler.yml:11]: same remark as above

File: .github/workflows/labeler.yml [hunk 3]
```diff
@@ -1,16 +1,12 @@
...
-name: labeler
-# Trigger the workflow on pull request events
+name: "Pull Request Labeler"
 on: [pull_request]
+
 jobs:
-  label:
-    runs-on: ubuntu-20.04
+  triage:
+    runs-on: ubuntu-latest
```
Comment (FrancescoBorzi) [inline .github/workflows/labeler.yml:6]: I think it's better to pin the version, so `ubuntu-20.04` in this case. So we have control on it.

File: .github/labeler.yml (hunk 1 above)
Comment (Kitzunu) [reply .github/labeler.yml:7]: Yes

File: .github/labeler.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+DB:
+  - src/**/*.sql
+
+CORE:
+  - src/**/*.cpp
+  - src/**/*.h
```
Comment (FrancescoBorzi) [inline .github/labeler.yml:6]: shouldn't we exclude `src/server/scripts/**` ? or the category `Script` will make this excluded?

**Review by FrancescoBorzi — APPROVED** [review]

File: .github/labeler.yml (hunk 2 above)
Comment (Kitzunu) [reply .github/labeler.yml:10]: actually you are right, it should be /**/*.cpp to include the subfolders too. My bad

**Pull request merged**